src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -342,8 +342,10 @@ export const StartDate: React.FC<Props> = ({
 }) => {
   const textKeys = useTextKeys()
   const [showError, setShowError] = React.useState(false)
+
+  // TODO: Make this flag more generic
   const isSingleStartDateBundle = (offerData: OfferData) =>
-    isBundle(offerData) && isDanish(offerData)
+    isBundle(offerData) && !isNorwegian(offerData)
```

Comment:
The new Accident SE bundle is meant to have one start date in the sidebar, the existing code shows two start dates. I generalised this statement so fix this.

Reply:
Wouldn't it be more explicit to write the check as `isDanish(offerData) || isSwedish(offerData)`?

Otherwise, I think a comment would be nice to explain why this logic makes sense - I feel like it's something easy to forget

This kind of business logic really should live somewhere else though :P